cos/static/game_page_api_calls.js

```diff
@@ -16,6 +16,7 @@ function buy_settlement (settlement_id, x, y, cbFunc) {
         success: function (data) {
             update_player_resources_table(data);
             cbFunc(x, y, data.player.player_color);
+            check_for_winner(end_game);
```

Comment:
Since start_turn() should be called right after this, and the start_turn() function also has a check_for_winner() call, this is probably not needed. But there is no harm in leaving it in at this point. So remove it only if you feel like it.

Reply:
I think this works better from a user experience standpoint as this way a player knows they've won immediately after placing the settlement.